npm/networkPolicyController_test.go

```diff
@@ -271,7 +272,8 @@ func TestAddMultipleNetworkPolicies(t *testing.T) {
 	// namedPort
 	netPolObj2.Spec.Ingress[0].Ports[0].Port = &intstr.IntOrString{StrVal: fmt.Sprintf("%s", netPolObj2.Name)}
 
-	f := newNetPolFixture(t)
+	fexec := exec.New()
```

Comment:
why cant we move this into newNetPolFixture itself, reducing the diff

Reply:
This is a temporary diff until this test suite is updated with fake exec. Until the suites are migrated, some tests are run with a real exec and some with a fake, so imo it's more pluggable to have the method of exec at test level